File: C++/숫자의표현.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

int solution(int n) {
    int answer = 0, add = 0;
    
    for(int i = 1 ; i < n+1 ; i++){
        add = 0;
        for(int j = i ; j < n+1 ; j++){
            add += j;
            if(add == n){answer++; break;}
            else if(add > n) break;
        }
    }
    
    return answer;
}

File: python/숫자의표현.py
import math

def solution(n):
    tmp = n
    answer = 0
    add = 0
    
    for i in range(1, n + 1):
        add = 0
        for j in range(i,n + 1):
            add += j
            if add == n:
                answer += 1
            elif add > n:
                break
        
    return answer